<?xml version="1.0" encoding="UTF-8" ?>

<launch>
    <node pkg="elic_calib" type="elic_calib_prog" name="elic_calib_prog_node" output="screen">
        <param name="config_filename"
               value="$(find elic_calib)/config/elic-config-real2.yaml"
               type="string"/>
        <param name="shot_screen_save_dir" value="$(find elic_calib)/scene" type="string"/>
    </node>
</launch>